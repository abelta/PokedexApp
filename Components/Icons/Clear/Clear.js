import React, { PureComponent } from 'react';
import { Image } from 'react-native';
import styles from './styles';

class Clear extends PureComponent {
  // setNativeProps(nativeProps) {
  //   this.root.setNativeProps(nativeProps);
  // }

  render() {
    return (
      <Image
        // ref={component => this.root = component}
        source={require('./cancel.png')} // eslint-disable-line global-require
        style={styles.clear}
        // {...this.props}
      />
    );
  }
}

export default Clear;
