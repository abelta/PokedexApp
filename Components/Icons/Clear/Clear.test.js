import React from 'react';
import renderer from 'react-test-renderer';
import Clear from '.';

describe('Clear', () => {
  const tree = renderer.create(<Clear />).toJSON();
  it('renders correctly', () => {
    expect(tree).toMatchSnapshot();
  });
});
