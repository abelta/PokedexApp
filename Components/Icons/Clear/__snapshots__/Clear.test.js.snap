// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Clear renders correctly 1`] = `
<Image
  source={1}
  style={
    Object {
      "height": 20,
      "width": 20,
    }
  }
/>
`;
